crates/bevy_ui/src/entity.rs

```diff
@@ -21,8 +25,22 @@ use bevy_transform::prelude::{GlobalTransform, Transform};
 pub struct NodeBundle {
     /// Describes the size of the node
     pub node: Node,
-    /// Describes the style including flexbox settings
-    pub style: Style,
+    /// The layout algorithm used
+    pub layout_strategy: LayoutStrategy,
+    /// The offset of this UI node, relative to its default position
+    pub offset: Offset,
+    /// Whether the node should be absolute or relatively positioned
+    pub position_type: PositionType,
+    /// The constraints on the size of this node
+    pub size_constraints: SizeConstraints,
+    /// The margin, padding and border of the UI node
+    pub spacing: Spacing,
+    /// The flexbox layout parameters
+    pub flex_layout: FlexLayout,
+    /// The direction of the text
+    pub text_direction: TextDirection,
```

Comment:
I don't think they should have it, but this was a direct port.

Reply:
I think we should actually keep it, because it affects the layout of the children in Flexbox. At some point we should probably make it possible to inherit the text direction and set that as default, so that you don't have to set it for each node separately.

Reply:
Does it do anything at all at the moment for those types of node? If it's not currently inherited and is therefore unused I think it should be removed. It can always be added later if/when such inheritance is actually implemented.

Reply:
From what I can tell it is not used for _anything_

Reply:
We probably first need to implement <https://github.com/DioxusLabs/taffy/issues/213>

Reply:
Yup. I say we remove it until we have solved https://github.com/DioxusLabs/taffy/issues/213, just so that it is clear we don't have it in just yet. Keep it around causes confusion